pkg/util/globalerror/errors.go

```diff
@@ -57,3 +60,7 @@ func (id ID) Message(msg string) string {
 func (id ID) MessageWithLimitConfig(flag, msg string) string {
 	return fmt.Sprintf("%s (%s%s). You can adjust the related per-tenant limit by configuring -%s, or by contacting your service administrator.", msg, errPrefix, id, flag)
 }
+
+func (id ID) MessageWith2LimitConfig(flag, flag2, msg string) string {
```

Comment:
I'm not a big fan of "With2" version. A cleaner alternative, from the caller perspective, may be reversing `MessageWithLimitConfig()` arguments order and making flags a variable argument, like this:

```
func (id ID) MessageWithLimitConfig(msg string, flags ...string) string {
```

Then, you could dynamically generate the error message to include all `flags`.

_I would do this change in a separate PR, to keep the change isolated from this PR._

Reply:
I changed the With2 to what you suggested, but kept flag as a separate string to ensure there's a compile error if there isn't at least 1 flag. After this is merged, I can open a separate PR to change all previous uses of the old one to use the new one and then remove the old one. Is this okay?

Reply:
The new function version LGTM!

> I can open a separate PR to change all previous uses of the old one to use the new one and then remove the old one.

LGTM